user_guide.adoc

```diff
@@ -1377,30 +1377,82 @@ library" for expression language.
 |Return type
 |Description
 
-|to_s
+|`to_s`
 |String
 |Converts integer into a string using decimal representation
-
-|to_s(radix)
```

Comment:
I couldn't find evidence in _BaseTranslator.scala_ that _to_s_ can accept arguments for type _IntType_.

Reference: https://github.com/kaitai-io/kaitai_struct_compiler/blob/add79599cc7d0a471b1908f4028ff0080a872f56/shared/src/main/scala/io/kaitai/struct/translators/BaseTranslator.scala#L114